quarkchain/cluster/tests/test_shard_state.py

```diff
@@ -116,42 +116,62 @@ def test_gas_price(self):
         env = get_test_env(
             genesis_account=acc_list[0],
             genesis_minor_quarkash=100000000,
-            genesis_minor_token_balances={"QKC": 100000000, "QI": 100000000},
+            genesis_minor_token_balances={
+                "QKC": 100000000,
+                "QI": 100000000,
+                "BTC": 100000000,
+            },
         )
 
         qkc_token = token_id_encode("QKC")
         qi_token = token_id_encode("QI")
+        btc_token = token_id_encode("BTC")
 
         state = create_default_shard_state(env=env)
 
         # Add a root block to have all the shards initialized
         root_block = state.root_tip.create_block_to_append().finalize()
         state.add_root_block(root_block)
 
-        # 5 tx per block, make 3 blocks
-        for nonce in range(3):
+        # 5 tx per block, make 7 blocks
```

Comment:
can keep use 5 blocks and update `state.gas_price_suggestion_oracle` for different testing scenarios